phpunit/class-wp-theme-json-test.php

```diff
@@ -4788,14 +4788,14 @@ public function test_get_shadow_styles_for_blocks() {
 		$this->assertSame( $expected_styles, $theme_json->get_stylesheet() );
 	}
 
-	public function test_get_top_level_background_image_styles() {
+	public function test_get_top_level_and_block_background_image_styles() {
```

Comment:
Note to self:

This is okay, but to specifically test block style output, I could test the output of `WP_Theme_JSON_Gutenberg::get_styles_for_block`, which is called (indirectly) by `WP_Theme_JSON_Gutenberg::get_stylesheet` anyway.

That means I keep `test_get_top_level_background_image_styles` and test top-level styles only, and create another test for blocks.
	
There's an effort to clean up tests, so let's wait for the outcome there:

- https://github.com/WordPress/wordpress-develop/pull/6734

Reply:
I'll wait until https://github.com/WordPress/gutenberg/pull/62463 is merged before rebasing and fixing these tests.